lib/logstash/outputs/scalyr.rb

```diff
@@ -306,7 +308,7 @@ def multi_receive(events)
         rescue Scalyr::Common::Client::ServerError, Scalyr::Common::Client::ClientError => e
           sleep_interval = sleep_for(sleep_interval)
           message = "Error uploading to Scalyr (will backoff-retry)"
-          exc_data = {
+          @exc_data = {
```

Comment:
exc data is class level variable, right? If so, do we need to synchronize it to avoid race conditions (aka another thread changing the value)?

Reply:
Arg you're right, it actually doesn't need to be class level to work so I'll change that rather than syncronizing.